rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+***NOTE**: DCP roadmaps will be a source for a shared understanding of DCP priorities in the future.*
+
+After this meeting, the **Product Owner** and **Technical Lead** for each component MUST ensure that the *Product Backlog* in their repo is refined and aligned to the DCP priorities.
+
+### Sprint Planning
+
+Prior to the start of the DCP Sprint, component teams identify their *Sprint Backlog* - a slice of items from the the ZenHub *Product Backlog* pipeline in their repo which are assigned to the ZenHub Milestone for the Sprint and moved to the ZenHub *Sprint Backlog* pipeline.
+
+The number of items selected is solely up to the Development Team. 
+- [ZenHub story points](https://help.zenhub.com/support/solutions/articles/43000034282) MUST be assigned to all *Sprint Backlog* items. (Story points are required to enable [ZenHub burndown charts](https://help.zenhub.com/support/solutions/articles/43000010356-track-sprint-progress-with-burndown-charts) to track the progress of a Sprint.)
+  - [How to start?](https://www.scrum.org/forum/scrum-forum/6774/how-decide-how-many-story-points-take-first-sprint)
+- Research and prototyping tasks SHOULD be modeled as Spikes.
+- Dependencies between components within a Sprint MUST be communicated between the respective teams and surfaced using [ZenHub Dependencies](https://help.zenhub.com/support/solutions/articles/43000010349). 
+
+At the start of the Sprint, the single DCP Sprint Backlog can be reviewed by **PM** and **Technical Architecture** in their oversight roles.
+
+### Sprint(ing)
+
+During Development, individual developers assign items from the ZenHub *Sprint Backlog* pipeline and move to the *In Progress* pipeline. 
+
+When there is a pull request in review for an *In Progress* item, the item is moved to the *QA/Review* pipeline. 
+
+If rejected, the item returns to *In Progress*; otherwise, it is deployed in Dev for verification and moves to *Done* pipeline.
+
+Items that are *Done* are moved to the *Closed* pipeline when verified in Dev or Integration. 
+
+During Sprints, **Technical Architecture** MAY schedule Scrums on a regular cadence for the DCP Development Team *"to identify if any integration issues exist."* (See *Daily Scrum* in [**Scrum Nexus**](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance)
+
+**PM** monitors progress using ZenHub charts and reports.
+
+### Sprint Review
+
+**Every four weeks**. On the penultimate day of the Sprint, there is a one hour meeting for the Sprint Review open to the entire DCP commmunity led by **PM**. Major product increments selected by **Technical Leads** are demonstrated to key stakeholders.
```

Comment:
This meeting will have to be strictly time-boxed. One hour is not much time if there are more than about 4-5 things to demo.  I know that we haven't had much to demo lately but we are all hoping that this changes.  We can always modify this to be 1.5 hours in the future.

Reply:
Noted. I think this could easily be addressed during the Retrospective process?